<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AGHPhone</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>28</y>
     <width>360</width>
     <height>485</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9" >
    <item>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Direct call:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Host address:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="hostAddressEdit" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Port:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="portEdit" />
       </item>
      </layout>
      <zorder>label</zorder>
      <zorder>hostAddressEdit</zorder>
      <zorder>label_2</zorder>
      <zorder>portEdit</zorder>
      <zorder>groupBox</zorder>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Recent Contacts:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="contactsList" />
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" >
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="callButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="toolTip" >
            <string>Call</string>
           </property>
           <property name="text" >
            <string>Call</string>
           </property>
           <property name="iconSize" >
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disengageButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="toolTip" >
            <string>Disengage</string>
           </property>
           <property name="text" >
            <string>Disengage</string>
           </property>
           <property name="iconSize" >
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>360</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDasf" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionZaloguj" />
    <addaction name="actionLog_out" />
    <addaction name="separator" />
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionOptions" />
    <addaction name="actionSearch_in_directory" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionHelp" />
    <addaction name="actionLicense" />
    <addaction name="actionAbout" />
    <addaction name="actionAbout_QT" />
   </widget>
   <addaction name="menuDasf" />
   <addaction name="menuTools" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>513</y>
     <width>360</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <action name="actionZaloguj" >
   <property name="text" >
    <string>Log in</string>
   </property>
  </action>
  <action name="actionLog_out" >
   <property name="text" >
    <string>Log out</string>
   </property>
  </action>
  <action name="actionEdit_profile" >
   <property name="text" >
    <string>Edit profile</string>
   </property>
  </action>
  <action name="actionOptions" >
   <property name="text" >
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="text" >
    <string>Help</string>
   </property>
  </action>
  <action name="actionLicense" >
   <property name="text" >
    <string>License</string>
   </property>
  </action>
  <action name="actionSearch_in_directory" >
   <property name="text" >
    <string>Search in directory</string>
   </property>
  </action>
  <action name="actionAbout_QT" >
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
